style.css

```diff
@@ -0,0 +1,162 @@
+.image1 {
+  width: 945px;
+  height: auto;
+}
+
+.image2 {
+  width: 600px;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.navbar {
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  position: fixed; /* Set the navbar to fixed position */
+  top: 0; /* Position the navbar at the top of the page */
+  width: 100%; /* Full width */
+  font-size: larger;
+  border-bottom: 1px solid black;
+  justify-items: baseline;
+}
+
+.nav {
+  flex: 1;
+  flex-wrap: wrap;
+  border: 5px solid transparent;
+}
+
+.nav1 {
+  flex: 0.15;
+}
+
+.nav2 {
+  flex: 0.15;
+}
```

Comment:
## Aim to remain DRY
- [ ] you appear to be repeating your self on lines `37`, & `#41`. Do you mind merging the rules to avoid this?